src/lib/time/__init__.js

```diff
@@ -40,10 +135,16 @@ var $builtinmodule = function (name) {
         return susp;
     });
 
+    mod.sleep.__doc__ = 
+        "sleep(seconds)\n" +
+        "\n" +
+        "Delay execution for a given number of seconds.  The argument may be\n" +
+        "a floating point number for subsecond precision.";
+
     function isLeapYear(year) {
         if((year & 3) != 0) return false;
         return ((year % 100) != 0 || (year % 400) == 0);
-    };
+    }
 
     function getDayOfYear(date) {
```

Comment:
I think you want to add an optional utc flag to this function.  Then from your gmtime function you can tell it to get you the day of the year using UTC.  I *think* this will fix your timezone bug and the test that is failing for @rixner back in CDT.